src/main/webapp/view/discussion/discussion.js

```diff
@@ -36,7 +36,11 @@ const SELECTOR_CANCEL_REPLY = '#cancel-reply';
 const SELECTOR_POST_REPLY = '#post-reply';
 const SELECTOR_REPLY_TEXTAREA = '#reply-textarea';
 
+// 10 seconds.
+const TIME_TOLERANCE_MILLISECONDS = 10000;
+
 let newCommentTimestampMs = 0;
+let /** !Array<DiscussionComment> */ currentRootDiscussionComments = [];
```

Comment:
Why do we have an inline comment here please?

Reply:
Type annotation for unclear variable type.  See: https://google.github.io/styleguide/jsguide.html#features-declare-types-as-needed